### PR TITLE
Update to bazel 5.4.1

### LINKED ISSUE #159: [bazel bug] mac编译spu报错
```
### Issue Type

Build/Install

### Modules Involved

SPU compiler

### Have you reproduced the bug with SPU HEAD?

Yes

### Installation Kind

source

### SPU Version

latest

### OS Platform and Distribution

macOS 13.0

### Python Version

_No response_

### Compiler Version

_No response_

### Current Behavior?

# macOS >= 11, Xcode >=13.0

# Install Xcode
https://apps.apple.com/us/app/xcode/id497799835?mt=12

# Select Xcode toolchain version
sudo xcode-select -s /Applications/Xcode.app/Contents/Developer

# Install homebrew
https://brew.sh/

# Install dependencies
# Be aware, brew may install a newer version of bazel, when that happens bazel will give an error message during build.
# Please follow instructions in the error message to install the required version
brew install bazel cmake ninja libomp go

# For Intel mac only
brew install nasm

# Install python dependencies
pip install -r requirements.txt
pip install -r requirements-dev.txt

```

基于文档以上步骤操作后，执行bazel build //... -c opt报错：
`ERROR: /private/var/tmp/_bazel_shuming/0087c1e03728cd0349dafefac9b798f6/external/local_config_python/BUILD:78:8: in outs attribute of genrule rule @local_config_python//:python_include: Genrules without outputs don't make sense
ERROR: /private/var/tmp/_bazel_shuming/0087c1e03728cd0349dafefac9b798f6/external/local_config_python/BUILD:78:8: Analysis of target '@local_config_python//:python_include' failed
ERROR: Analysis of target '//spu/utils:distributed' failed; build aborted: 
INFO: Elapsed time: 0.398s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded, 0 targets configured)
    currently loading: @bazel_tools//src/tools/launcher
    Fetching @llvm_terminfo; Restarting.
`

求教，感谢

### Standalone code to reproduce the issue

```Python
如题
```

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by anakinxc**
Fixed #159

### CODE REVIEW

File: .bazelversion [hunk 1]
```diff
@@ -1 +1 @@
-5.4.0
+5.4.1
```
Comment (tpppppub) [inline .bazelversion:1]: Document needs to be changed too.

**Review by tpppppub — CHANGES_REQUESTED** [review]

File: .bazelversion (hunk 1 above)
Comment (anakinxc) [reply .bazelversion:1]: Done

**Review by 6fj — APPROVED** [review]
lgtm

**Review by tpppppub — APPROVED** [review]
LGTM

**Pull request merged**